website/static/js/fangorn.js

```diff
@@ -2441,9 +2439,7 @@ function isInvalidDropItem(folder, item, cannotBeFolder, mustBeIntra) {
         // no dropping if waiting on waterbutler ajax
         item.inProgress ||
         (cannotBeFolder && item.data.kind === 'folder') ||
```

Comment:
Should you be able to drag folders into figshare now? If so, this line prevents that.